lisa/tools/git.py

```diff
@@ -47,10 +47,13 @@ def clone(
         ref: str = "",
         dir_name: str = "",
         fail_on_exists: bool = True,
+        recurse_submodules: bool = False,
```

Comment:
It looks doesn't need an. Let's just bring this in command line always.

Reply:
Please remove this argument too.

Reply:
Removed, will squash when I merge